packages/plugin-evm/package.json

```diff
@@ -11,7 +11,7 @@
         "@lifi/sdk": "3.4.1",
         "@lifi/types": "16.3.0",
         "tsup": "8.3.5",
-        "viem": "2.21.53"
+        "viem": "2.21.58"
```

Comment:
We should probably do ^2.21.58 for both so it auto updates based off of latest changes for both